packages/liveblocks-react-ui/src/components/AiChat.tsx

```diff
@@ -178,6 +178,13 @@ export const AiChat = forwardRef<HTMLDivElement, AiChatProps>(
     }
     const scrollToBottom = scrollToBottomCallbackRef.current;
 
+    useEffect(() => {
```

Comment:
It feels a bit complex to have this here and keep the same `scrollToBottom("instant")` call in `AutoScrollHandler`, maybe we should remove `AutoScrollHandler` and bring everything here.

This effect would get a dependency of `Boolean(messages)` and would handle both `scrollToBottom("instant")` cases.